src/picosvg/geometric_types.py

```diff
@@ -50,6 +50,11 @@ def __add__(self, other: "Vector") -> "Point":
     def round(self, digits: int) -> "Point":
         return Point(round(self.x, digits), round(self.y, digits))
 
+    def tolerent_equals(
+        self, other: "Point", tolerence=_DEFAULT_ALMOST_EQUAL_TOLERENCE
+    ) -> bool:
+        return abs(self.x - other.x) <= tolerence and abs(self.y - other.y) <= tolerence
```

Comment:
You could use `math.isclose` with given `rel_tol` parameter, it's supposed to do exactly that
https://docs.python.org/3/library/math.html#math.isclose

Reply:
Maybe you decided to not use math.isclose because of its handling of comparison near zero (I see in your test case you are starting the path at M0,0). The math.isclose function has a distinct abs_tol parameter for the absolute tolerance (defaulting to 0.0).
Check out the PEP485 (which introduced math.isclose) on why relative comparison is problematic if either value is zero: https://www.python.org/dev/peps/pep-0485/#behavior-near-zero

Reply:
`math.isclose(self,x, other.x, abs_tol=tolerence)` doesn't seem like it's gaining me anything. I don't mind changing it if you prefer it but I actually like this way better.

Reply:
Absent active protest I'm going to leave it this way. Resolving, please re-ping if you disagree.